data/internship.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable max-len */
+
+import React from 'react';
+import styles from '../styles/pages/Internship.module.scss';
+
+export default {
+    items: [
+        {
+            date: '9/27',
+            name: 'Fall General Meeting',
+            info: 'Come learn about the different ACM committees!',
+            active: true,
+        },
+        {
+            date: '10/10',
+            name: 'Application Form is Open',
+            info: 'Fill out the application form for up to 3 committees that you would like to intern for',
+            active: false,
+        },
+        {
+            date: '10/22',
+            name: 'Applications Due',
+            info: 'Submit your application by 11:59 pm',
+            active: false,
+        },
+        {
+            date: '10/22-11/02',
+            name: 'Interviews',
+            info: 'A handful of applicants will be chosen for an interview with the committee(s) you applied to',
+            active: false,
+        },
+        {
+            date: '11/03-11/04',
+            name: 'Decisions Released',
+            info: 'Hear back via Email what committee(s) accepted you (if you applied to multiple, decide which one you would like to intern for)',
+            active: false,
+        },
+        {
+            date: '11/06',
+            name: 'Official ACM Onboarding',
+            info: 'Learn what you need to know as an ACM Intern',
+            active: false,
+        },
+    ],
+    testimonials: [
+        {
+            name: 'Maya Raman',
+            quote: <>
```

Comment:
All of these should be `<h3>`s in hierarchy. Alternatively, restyle and make the intern name an `<h3>` (semantically makes more sense).

Reply:
im not sure why the name being a h3 makes more sense - since it isn't serving as a header
resetting them to h3 might be a better optio